templates/etc_dnsmasq.conf.j2

```diff
@@ -69,6 +69,8 @@ server={{ dnsmasq_upstream_servers }}
 
 {% if dnsmasq_authoritative %}
 dhcp-authoritative
+# Set auth-server so we also get the zone SOA serial set by timestamp
+auth-server={{ dnsmasq_domain }},{{ dnsmasq_interface }}
```

Comment:
I'm curious if using the `mandatory` filter on these two parameters could help ensure they are set, or if it would  trip things up if `dnsmasq_authoritative` is false and they are unset.
```suggestion
auth-server={{ dnsmasq_domain|mandatory }},{{ dnsmasq_interface|mandatory }}
```